scripts/generate-version-info.sh

```diff
@@ -13,11 +13,19 @@ fi
 OUTPUT_FILE="$1"
 
 # packages
-sudo rpm --query --all --queryformat '\{"%{NAME}": "%{VERSION}-%{RELEASE}"\}\n' | jq --slurp --sort-keys 'add | {packages:(.)}' > "$OUTPUT_FILE"
+sudo rpm --query --all --queryformat '\{"%{NAME}": "%{VERSION}-%{RELEASE}"\}\n' | jq --slurp --sort-keys 'add | {packages:(.)}' > "${OUTPUT_FILE}"
+
+# version detection
+kubelet_version=$(kubelet --version | cut -d' ' -f2)
+awscli_version=$(aws --version | cut -d' ' -f1 | cut -d'/' -f2)
```

Comment:
to avoid shellcheck mistakenly take `$1` and `$2` in awk as warning, turns to use `cut -d' ' -f2` instead.